doc/README.md

```diff
@@ -136,9 +136,15 @@ Panning is special because it's accessible in any mode (if you have a keyboard)
 by holding down the <kbd>Space</kbd> key and then dragging the page.
 (This behavior matches Adobe Creative Suite.)
 
+A mouse scroll-wheel or a touchpad's scroll behavior (typically
+two-finger dragging) will also pan the canvas.
```

Comment:
Perhaps mention "no matter what mode you're in". Also, no hyphen in "scroll wheel" (several times). Maybe "scroll wheel" would be better than "mouse scroll wheel" &mdash; there are a lot of devices that have scroll wheels. Finally, I think "scroll gesture" would be more precise than "scroll behavior" (several times, also "zoom gesture").

Reply:
Yes, gesture definitely sounds better! Hyphens gone, 'mouse' modifier gone.